src/components/notifications/noti-tray.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, h, Method, State } from "@stencil/core"
+
+@Component({
+  tag: "oui-noti-tray",
+  styleUrl: "noti-tray.css",
+})
+export class NotiTray {
+  @State() private showNoti = false
+
+  public componentDidLoad() {
+    setInterval(this.demoImport.bind(this), 5000)
+  }
+
+  public componentDidUnload() {
+    clearInterval(this.demoImport.bind(this))
+  }
+
+  @Method()
+  public open() {
+    switch (this.showNoti) {
+      case false:
+        this.showNoti = true
+        document.querySelector("oui-noti-tray").style.display = "block"
+        break
+      default:
+        this.showNoti = false
+        document.querySelector("oui-noti-tray").style.display = "none"
+        break
+    }
+  }
+
+  public render() {
+    return (
+      <aside>
+        <header>
+          <button
+            class="oui-noti-tray__clear"
+            onClick={this.onClearNoti.bind(this)}
+          >
+            Clear All
+          </button>
+        </header>
+        <section class="oui-noti-tray__statuses">
+          <ul class="oui-noti-tray__statusList"></ul>
+        </section>
+      </aside>
+    )
+  }
+
+  private demoImport() {
```

Comment:
This component should implement a API to push the actual notification data. This demo function is a mock function and not an actual implementation. The notifications should come from a data source outside the component (a prop or a method) and the component should react to the changes of the prop.